eng/pipelines/OpenApiSpecs/repo-structure-cache-job.yml

```diff
@@ -0,0 +1,39 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+parameters:
+- name: DB_NAME
+  type: string
+  default: "openapi-portal"
+
+- name: COLLECTION_NAME
+  type: string
+  default: "repoStructureCache"
+
+- name: REPOS_URL_LIST
+  type: string
+  default: "https://github.com/test-repo-billy/azure-rest-api-specs/tree/main,https://github.com/test-repo-billy/azure-rest-api-specs-pr/tree/RPSaaSDev,https://github.com/test-repo-billy/azure-rest-api-specs-pr/tree/main,https://github.com/test-repo-billy/azure-rest-api-specs/tree/RPSaaSDev"
+
+trigger:
+- main
+
+pool:
+  vmImage: ubuntu-latest
```

Comment:
We should use our image pools. Example https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/githubio-linkcheck.yml#L5-L6